### ISSUE TITLE
I use Vue sfc, ts-loader will check html and css, but I don't want that

### ISSUE THREAD

**Issue by martinko2009** [palantir/tslint#4403]
### Bug Report

- __TSLint version__: 5.11.0
- __TypeScript version__: 3.1.6
- __Running TSLint via__: (pick one) webpack

#### TypeScript code being linted
I use Vue SFC to develop, but I find that ts-loader will throw checking errors in html and css. I think ts-loader check the html in tag and css in <style> tag, but I don't want it to check html and css, just only check ts is good.
```ts
<template>
	<div>
        <div>欢迎您[<span class="visitor" v-text="marks"></span>]
            来到Typescript的世界</div>
        <button @click="changed">换个人</button>
    </div>
</template>
<script lang='ts'>
class Test {
    names: Array<string> = ['小王', '小强', '小张'];
    idx = 0;
    GetVisitor (): string {
        this.idx++;
        if (this.idx === this.names.length) {
            this.idx = 0;
        }
        return this.names[this.idx];
    }

};
const tester = new Test();
import Vue from 'vue';
import Component from 'vue-class-component';

@Component
export default class HelloHello extends Vue {
    //data
	marks = tester.GetVisitor();

    //methods
	changed () {
		this.marks = tester.GetVisitor();
    }
}
</script>
```
**webpack.config.js**
```
{
	enforce: "pre",
	test: /\.(vue)$/,
	include:[path.resolve(__dirname,"./src"), path.resolve(__dirname,"./assets")],
	exclude: [path.resolve(__dirname,"./node_modules/"), path.resolve(__dirname,"./assets/components/skeleton")],
	use: [
		{
			loader: 'tslint-loader',
			options: { 
			}
		}
		]
},
{
	test: /\.vue$/, 
	loader: 'vue-loader',
		  options: {
			  loaders:{
				  js: [{
					  loader: 'babel-loader',
					  options: {
						  compact: false,
						  cacheDirectory: true,
						  presets: ['@babel/preset-env'],
						  plugins: ['@babel/plugin-transform-runtime','@babel/plugin-proposal-object-rest-spread',["@babel/plugin-proposal-decorators", { "legacy": true }]]
						  }
					  }],
					  ts: "ts-loader",
					  tsx: "babel-loader!ts-loader",
				      css: [MiniCssExtractPlugin.loader, 'css-loader'], 
					  scss: [MiniCssExtractPlugin.loader, 'css-loader', 'sass-loader'],
					  extractCSS:true
				  }
			  }
		  },
```
with `tslint.json` configuration:

```json
{
    "rulesDirectory": [
        "./node_modules/tslint-eslint-rules/dist/rules"
    ],
    "jsRules": {
        "no-empty": true
    },
    "rules": {
        // Rules from tslint-microsoft-contrib
        // "function-name": [true, {
        //     "method-regex": "^[a-z]\\w+$",
        //     "private-method-regex": "^_[a-z]\\w+$",
        //     "protected-method-regex": "^[a-z_]\\w+$",
        //     "static-method-regex": "^[a-z]\\w+$",
        //     "function-regex": "^\\w+$"
        // }],
        // "no-delete-expression": true,
        // "no-document-domain": true,
        // "no-disable-auto-sanitization": true,
        // "no-exec-script": true,
        // "no-function-constructor-with-string-args": true,
        // "no-string-based-set-immediate": true,
        // "no-string-based-set-interval": true,
        // "no-string-based-set-timeout": true,
        // "react-no-dangerous-html": true,
        // "react-this-binding-issue": [true, { "allow-anonymous-listeners": true }],
        // "no-inner-html": true,
        // "no-document-write": true,

        // Rules that are built in to tslint
        //"align": [true, "statements"],
        "class-name": true,
        "curly": true,
        "eofline": true,
        "forin": true,
        "indent": [true, "tabs", 4],
        "interface-name": [true, "never-prefix"],
        "label-position": true,
        "max-line-length": [true, 140],
        "member-access": [true, "no-public"],
        "new-parens": true,
        "no-arg": true,
        "no-bitwise": false,
        "no-console": [true,
            "debug",
            "info",
            "time",
            "timeEnd",
            "trace"
        ],
        "no-consecutive-blank-lines":  true,
        "no-conditional-assignment": true,
        "no-construct": true,
        "no-debugger": true,
        "no-duplicate-variable": true,
        "no-empty": true,
        "no-duplicate-super": true,
        "no-duplicate-switch-case": true,
        "no-eval": true,
        "no-inferrable-types": true,
        "no-invalid-this": true,
        "no-string-throw": true,
        "no-switch-case-fall-through": true,
        "no-trailing-whitespace": true,
        "no-unused-expression": false,
        "no-var-keyword": true,
        "prefer-const": true,
        "no-string-literal": true,
        "no-sparse-arrays": true,
        "ban-comma-operator": true,
        "use-isnan": true,
        "linebreak-style": [true, "CRLF"],
        "comment-format": true,
        "no-boolean-literal-compare": true,
        "no-unnecessary-initializer": true,
        "space-before-function-paren": [true, "always"],
        "trailing-comma": [
            true,
            {
                "multiline": "never",
                "singleline": "never",
                "esSpecCompliant": true
            }
        ],
        "arrow-parens": [true, "ban-single-arg-parens"],
        "one-line": [true,
            "check-open-brace",
            "check-catch",
            "check-finally",
            "check-else",
            "check-whitespace"
        ],
        "quotemark": [true, "single", "jsx-single"],
        "radix": true,
        "semicolon": true,
        "triple-equals": true,
        "whitespace": [true,
            "check-branch",
            "check-decl",
            "check-operator",
            "check-module",
            "check-separator",
            "check-type",
            "check-rest-spread",
            "check-type-operator",
            "check-preblock"
        ],
        "arrow-return-shorthand": true,
        "no-angle-bracket-type-assertion": true,
        "one-variable-per-declaration": true,
        "no-object-literal-type-assertion": true,
        "no-unnecessary-type-assertion": true,
        "no-duplicate-imports": true,
        "import-spacing": true,
        "type-literal-delimiter": true,
        "no-floating-promises": [true],
        "file-header": [true],
        "ordered-imports": [false,
            {
                "import-sources-order": "case-insensitive",
                "module-source-path": "basename",
                "named-imports-order": "case-insensitive",
                "grouped-imports": true
            }
        ]
    }
}
```
**error information**
```
./src/demo/type-script/hello/hello.vue?vue&type=template&id=183bd9b1& (./node_modules/vue-loader/lib/loaders/templateLoader.js??vue-loader-options!./node_modules/vue-loader/lib??vue-loader-options!./node_modules/tslint-loader??ref--1-0!./src/demo/type-script/hello/hello.vue?vue&type=template&id=183bd9b1&)
Module Warning (from ./node_modules/tslint-loader/index.js):
[3, 30]: " should be '
[3, 47]: " should be '
[5, 24]: " should be '
[3, 29]: Missing semicolon
[3, 39]: Missing semicolon
[3, 46]: Missing semicolon
[3, 63]: Missing semicolon
[5, 16]: Missing semicolon
[8, 8]: Missing semicolon
[35, 10]: Missing semicolon
[3, 41]: missing whitespace
[3, 42]: missing whitespace
[3, 54]: missing whitespace
[3, 55]: missing whitespace
[5, 10]: missing whitespace
[4, 28]: missing whitespace
[4, 29]: missing whitespace
[8, 2]: missing whitespace
[7, 2]: missing whitespace
[6, 6]: missing whitespace
[5, 37]: missing whitespace
[5, 38]: missing whitespace
[5, 33]: missing whitespace
[5, 34]: missing whitespace
[8, 13]: missing whitespace
[8, 14]: missing whitespace
[8, 18]: missing whitespace
[3, 30]: missing whitespace
[3, 47]: missing whitespace
[5, 24]: missing whitespace
[1, 1]: Type assertion using the '<>' syntax is forbidden. Use the 'as' syntax instead.
[3, 18]: Type assertion using the '<>' syntax is forbidden. Use the 'as' syntax instead.
[3, 55]: Type assertion using the '<>' syntax is forbidden. Use the 'as' syntax instead.
[35, 1]: Type assertion using the '<>' syntax is forbidden. Use the 'as' syntax instead.
[1, 1]: missing file header
```

**Comment by johnwiseheart** [comment]
Closing as duplicate of https://github.com/palantir/tslint/issues/4379. This is a hard issue to solve, Vue single file components don't work with tslint right now.
